docs/source/conf.py

```diff
@@ -79,7 +81,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns: str = []
```

Comment:
Should this not rather be `list[str]` instead of `str`?

Reply:
Yes.  Thanks.  Fixing.